src/reference/00-Getting-Started/07B-Appending-Values.md

```diff
@@ -0,0 +1,84 @@
+---
+out: Appending-Values.html
+---
+
+Appending values
+----------------
+
+### Appending to previous values: `+=` and `++=`
+
+Assignment with `:=` is the simplest transformation, but keys have other
+methods as well. If the `T` in `SettingKey[T]` is a sequence, i.e. the key's
+value type is a sequence, you can append to the sequence rather than
+replacing it.
+
+- `+=` will append a single element to the sequence.
+- `++=` will concatenate another sequence.
+
+For example, the key `sourceDirectories in Compile` has a `Seq[File]` as its
```

Comment:
is `sourceDirectories in Compile` a key? 

Or is `sourceDirectories in Compile` a task and `sourceDirectories` the key? 

If `sourceDirectories in Compile` is a task, is the un-scoped `sourceDirectories` also a task?

This is not clear to me, even given this description, as someone who has used SBT for years. We need to be more precise and consistent with this terminology so we can drill into people the same set of standard terms